support-frontend/assets/pages/[countryGroupId]/router.tsx

```diff
@@ -4,6 +4,7 @@ import { setUpTrackingAndConsents } from 'helpers/page/page';
 import { renderPage } from 'helpers/rendering/render';
 import { geoIds } from 'pages/geoIdConfig';
 import { Checkout } from './checkout';
+import { GuardianLightLanding } from './guardianLightLanding';
```

Comment:
Something I realised this week - the file name was initially supposed to mirror the routes. Hence why `thank-you.ts` is spelled that way. I didn't spot this when I named the `oneTimeCheckout`, not sure if we want to use this convention?

Reply:
Don't we have a `threeTier**Landing**` route that comes from `contribute` however? Thats what I followed here as you know its a Landing Page that way.

Reply:
I agree that's a better pattern as you get more from the file name than the whole file path route. Just something to think about!